packages/langium/src/lsp/completion/completion-provider.ts

```diff
@@ -442,6 +441,11 @@ export class DefaultCompletionProvider implements CompletionProvider {
         });
     }
 
+    protected isKeywordWorthyToPropose(context: CompletionContext, keyword: ast.Keyword): boolean {
```

Comment:
Minor: Sounds a bit verbose and it probably makes sense to keep this more in line with the existing `filterCrossReference`.
```suggestion
    protected filterKeyword(context: CompletionContext, keyword: ast.Keyword): boolean {
```

Reply:
done